NEWS

```diff
@@ -57,7 +67,8 @@ issues]].
 None
 
 *** Removed
-None
+- he.tbl -> superseded by he-IL-comp8.utb
```

Comment:
We normally don't mention ".tbl" tables in the NEWS file.

Reply:
I removed it